openlibrary/templates/books/check.html

```diff
@@ -53,7 +53,7 @@ <h3 class="booktitle"><a href="javascript:;" title="$_('Select this book')">$wor
 </div>
 
 <script type="text/javascript">
-<!--
+<--
```

Comment:
Why was this change made? (Revert if not necessary; I think this will cause some issues.)